skimage/filters/_gaussian.py

```diff
@@ -155,3 +155,136 @@ def _guess_spatial_dimensions(image):
+    band-pass filters to multi-dimensional arrays. The input array is
+    blurred with two Gaussian kernels of differing sigmas to produce two
+    intermediate, filtered images. The more-blurred image is then subtracted
+    from the less-blurred image. The final output image will therefore have
+    had high frequency components attenuated by the smaller-sigma Gaussian, and
+    low frequency components will have been removed due to their presence in
+    the more-blurred intermediate.
+
+    Parameters
+    ----------
+    image : ndarray
+        Input array to filter.
+    low_sigma : scalar or sequence of scalars
+        Standard deviation(s) for the Gaussian kernel with the smaller sigmas
+        across all axes. The standard deviations are given for each axis as a
+        sequence, or as a single number, in which case the single number is
+        used as the standard deviation value for all axes.
+    high_sigma : scalar or sequence of scalars, optional (default is None)
+        Standard deviation(s) for the Gaussian kernel with the larger sigmas
+        across all axes. The standard deviations are given for each axis as a
+        sequence, or as a single number, in which case the single number is
+        used as the standard deviation value for all axes. If None is given
+        (default), sigmas for all axes are calculated as 1.6 * low_sigma.
+    mode : {'reflect', 'constant', 'nearest', 'mirror', 'wrap'}, optional
+        The ``mode`` parameter determines how the array borders are
+        handled, where ``cval`` is the value when mode is equal to
+        'constant'. Default is 'nearest'.
+    cval : scalar, optional
+        Value to fill past edges of input if ``mode`` is 'constant'. Default
+        is 0.0
+    multichannel : bool, optional (default: False)
+        Whether the last axis of the image is to be interpreted as multiple
+        channels. If True, each channel is filtered separately (channels are
+        not mixed together).
+    truncate : float, optional (default is 4.0)
+        Truncate the filter at this many standard deviations.
+
+    Returns
+    -------
+    filtered_image : ndarray
+        the filtered array.
+
+    See also
+    --------
+    skimage.feature.blog_dog
+
+    Notes
+    -----
+    This function will subtract an array filtered with a Guassian kernel
+    with sigmas given by ``high_sigma`` from an array filtered with a
+    Gaussian kernel with sigmas provided by ``low_sigma``. The values for
+    ``high_sigma`` must always be greater than or equal to the corresponding
+    values in ``low_sigma``, or a ``ValueError`` will be raised.
+
+    When ``high_sigma`` is none, the values for ``high_sigma`` will be
+    calculated as 1.6x the corresponding values in ``low_sigma``. This ratio
+    was originally proposed by Marr and Hildreth (1980) [1]_ and is commonly
+    used when approximating the inverted Laplacian of Guassian, which is used
```

Comment:
```suggestion
    used when approximating the inverted Laplacian of Gaussian, which is used
```

Reply:
It seems my fingers really wanted to misspell that word